ds-controller-board/res/configuration.toml

```diff
@@ -83,7 +83,7 @@ PublishTopicPrefix = "edgex/events/device" # /<device-profile-name>/<device-name
 [DriverConfig]
   DeviceName = "controller-board"
   VID = "2341"
-  PID = "8037"
+  PID = "0037"
```

Comment:
For consistency these should be `numbers` , not strings, like they are in Card Reader. Refactor in a separate PR.

Reply:
This one is a bit unique because the card reader and controller board are different types of usb devices so the structs and types are very different. The card reader is using an gvalkov InputDevice which uses uint16 while the controller-board uses serial.v1 PortDetails which uses strings.

Reply:
Ahhh, ok thanks for the clarification.